src/Symfony/Component/HttpClient/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 ---
 
  * Add support for amphp/http-client v5 on PHP 8.4+
+ * Add IPv6 support to NativeHttpClient
```

Comment:
that's gonna be for 7.3 now

Reply:
Rebased my branch and changed PR's base branch